pkg/volume/fc/fc.go

```diff
@@ -277,7 +277,7 @@ func (plugin *fcPlugin) ConstructBlockVolumeSpec(podUID types.UID, volumeName, m
 	if err != nil {
 		return nil, err
 	}
-	klog.V(5).Infof("globalMapPathUUID: %v, err: %v", globalMapPathUUID, err)
+	klog.V(5).InfoS("GlobalMapPathUUID", "globalMapPathUUID", globalMapPathUUID, "err", err)
```

Comment:
There should not be an error here

Reply:
Yeah, you're right. This func will be already returned if `FindGlobalMapPathUUIDFromPod` returned err. I have refresh it, waitiing for your review with pleasure. @msau42